src/sider/arbeidsuforhet/ForhandsvarselBeforeDeadline.tsx

```diff
@@ -19,7 +19,7 @@ const texts = {
     "Dersom du har mottatt nye opplysninger og vurdert at bruker likevel oppfyller § 8-4, klikker du på Oppfylt-knappen. Du kan ikke avslå før fristen er gått ut.",
   frist: "Fristen går ut: ",
   oppfylt: "Oppfylt",
-  avslag: "Avslag",
+  avslag: "Innstilling om avslag",
```

Comment:
Jeg endret navnet på denne avskrudde knappen også, så det er samme tekst som kommer når fristen har gått ut. I nyeste versjon av skissene er det kun "Oppfylt"-knapp uten en avskrudd "avslag"-knapp, men det kan endres for seg selv.